spdx/writers/rdf.py

```diff
@@ -1016,10 +1016,11 @@ def write(self):
         for package_node in self.packages():
             package_triple = (doc_node, self.spdx_namespace.describesPackage, package_node)
             self.graph.add(package_triple)
-        """# Add relationship
-        relate_node = self.relationships()
-        relate_triple = (doc_node, self.spdx_namespace.relationship, relate_node)
-        self.graph.add(relate_triple)"""
+        # Add relationship
+        for relate_node in self.relationships():
+            relate_triple = (doc_node, self.spdx_namespace.relationship, relate_node)
+            self.graph.add(relate_triple)
+
```

Comment:
I did the same as in your comment [here](https://github.com/spdx/tools-python/pull/207#discussion_r964963555) and still got the same result. I think this should be fixed before merging this PR.